rmgpy/kinetics/arrhenius.pyx

```diff
@@ -249,7 +249,23 @@ cdef class Arrhenius(KineticsModel):
 
         # Set the rate parameter to a cantera Arrhenius object
         ctReaction.rate = self.toCanteraKinetics()
-    
+
+    cpdef ArrheniusEP toArrheniusEP(self):
```

Comment:
Yeah. I can set an alpha option, though I am hesitant to add features that may never be used. Other places in the code that convert Arrhenius to ArrheniusEP set alpha equal to 0.

Do you know when/how we would use the extra parameter?